urdf/include/urdf/model.h

```diff
@@ -51,10 +51,10 @@ namespace urdf{
 class Model: public ModelInterface
 {
 public:
-  /// \brief Load Model from TiXMLElement
-  bool initXml(TiXmlElement *xml);
-  /// \brief Load Model from TiXMLDocument
-  bool initXml(TiXmlDocument *xml);
```

Comment:
Removing these constructors in kinetic would break API/ABI compatibility. This change would have to wait until the next Ros release (ROS M).